dist/openshift/cincinnati-deployment.yaml

```
@@ -189,7 +189,7 @@ objects:
           metadata:
             serverAddress: http://prometheus-app-sre.openshift-customer-monitoring.svc.cluster.local:9090
             metricName: cincinnati_policy_engine_graph_incoming_requests_rate
-            threshold: "${{PE_REQ_AVG}}"
+            threshold: "${PE_REQ_AVG}"
```

Would the following be preferable for consistency with how we use e.g. MAX_REPLICAS/MIN_REPLICAS above on L185-186?

```suggestion
            threshold: ${{PE_REQ_AVG}}
```

---

we moved to `"${{PE_REQ_AVG}}"` in https://github.com/openshift/cincinnati/pull/956 because `${{PE_REQ_AVG}}` wont work. 
The actual solution according to app-sre is `"${PE_REQ_AVG}"`

---

:+1: